tests/plugins/filters.test.ts

```diff
@@ -218,6 +218,39 @@ describe("Filters plugin", () => {
       setCellContent(model, "A4", "Something");
       expect(zoneToXc(model.getters.getFilterTables(sheetId)[0].zone)).toEqual("A1:B3");
     });
+
+    test("Adjacent cells selection while applying filter on single cell", async () => {
```

Comment:
This test only test that your helper (`createFilter`) is correct. If you comment your change in the file `top_bar.ts`, this test will still pass.
What you can do is to test the UI (so in another file, see my previous comment). You can check the file `top_bar.test.ts`, there are some tests of the filter tool. So you can write a test, with the same data as in this test, start with `selectCell(model, "A1")` and then click on the button, and check two things: 
- Your selection is updated
- The created filter has the correct zone.

Reply:
Added the new test, let me know if you need any changes

Reply:
So this test can be removed, it's already test in the newly added test

Reply:
Remove the old test